src/components/DataTable/DataTimelineHeader/DataTimelineHeader.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { TimelineRange } from "../../../services/projection";
+import { Flex } from "@chakra-ui/layout";
+import { Center, Text } from "@chakra-ui/react";
+import { format } from "date-fns";
+import { formatDateToUTC } from "../../../services/helpers/utcdate";
+
+type DataTimelineHeaderProps = {
+  heading: string;
+  timeline: TimelineRange[];
+};
+
+function DataTimelineHeader({
+  heading,
+  timeline,
+}: DataTimelineHeaderProps): JSX.Element {
+  return (
+    <Flex py={6} pl={11} flexDirection="row">
+      <Flex width={200}>{heading}</Flex>
+      {timeline.map(({ title, type, startDate }, index) => {
+        return (
+          <Flex
+            flex={1}
+            fontFamily="Inter"
+            color="#333333"
+            fontWeight="700"
+            letterSpacing="0.05em"
+            key={title}
+          >
+            <Center
+              flex={1}
+              key={index}
+              alignItems={"start"}
+              flexDirection="column"
+            >
+              <Center height={4} flex={1}>
+                <Text textStyle="tableHead" fontSize="12px">
+                  {title}
+                </Text>
+              </Center>
+
+              {/* Sub Heading */}
```

Comment:
Are we leaving this comment in place?

Reply:
These can be removed, it was just for organization purposes.